data/account.go

```diff
@@ -1,6 +1,12 @@
 package data
 
-// Account defines the data structure for an account
+// AccountModel defines an account model (with associated information)
+type AccountModel struct {
+	Account   Account
+	BlockInfo BlockInfo
+}
+
+// AccountResource defines the data structure for an account
```

Comment:
mismatch between struct name and comment

Reply:
Fixed.